### PR TITLE
CLDR-14804 add an API for calculating a historic vetting report

### PR DESCRIPTION
**Pull request by srl295**
CLDR-14804

- [ ] This PR completes the ticket.

### CODE REVIEW

File: tools/cldr-apps/js/src/views/VettingParticipation2.vue [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    };
+  },
+  // Note: note loading fetchData()  on create() due to slowness
+  methods: {
+    async fetchData() {
+      try {
+        this.loading = true;
+        const raw = await fetch(`api/voting/participation`, {
+          headers: cldrStatus.sessionHeaders(),
+        });
```
Comment (srl295) [inline tools/cldr-apps/js/src/views/VettingParticipation2.vue:37]: could be abstracted by doFetch eventually

**Comment by btangmu** [comment]
"getInterestLocales in class org.unicode.cldr.web.UserRegistry.User cannot be applied to given types"

**Comment by srl295** [comment]
> "getInterestLocales in class org.unicode.cldr.web.UserRegistry.User cannot be applied to given types"

i see it on the build server but not locally, does not make sense though…

**Comment by srl295** [comment]
> > "getInterestLocales in class org.unicode.cldr.web.UserRegistry.User cannot be applied to given types"
> 
> i see it on the build server but not locally, does not make sense though…

it was due to a merge conflict apparently, changes to UserRegistry in your other PR… i added some comments

**Comment by srl295** [comment]
added a link
image

File: tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/VotingParticipation.java [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(
+        summary = "Get voting participation summary",
+        description = "Get voting participation summary")
+    @APIResponses(
+        value = {
+            @APIResponse(
+                responseCode = "200",
+                description = "Vote participatoin results",
+                content = @Content(mediaType = "application/json",
```
Comment (btangmu) [inline tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/VotingParticipation.java:54]: typo: "participatoin"

**Review by btangmu — APPROVED** [review]

**Pull request merged**